src/info/repo/author.rs

```diff
@@ -135,4 +135,80 @@ mod test {
 
         assert_eq!(author.to_string(), "75% John Doe 1500");
     }
+
+    #[test]
+    fn test_author_info_title() {
+        let author = Author::new(
+            "John Doe".into(),
+            "john.doe@email.com".into(),
+            1500,
+            2000,
+            true,
+        );
+
+        let author2 = Author::new(
+            "Roberto Berto".into(),
+            "bertolone2000@email.com".into(),
+            240,
+            300,
+            false,
+        );
+
+        let authors_info = AuthorsInfo {
+            info_color: DynColors::Rgb(255, 0, 0),
+            authors: vec![author.clone()],
+        };
+
+        let authors_info_2 = AuthorsInfo {
+            info_color: DynColors::Rgb(255, 0, 0),
+            authors: vec![author2, author],
+        };
+
+        assert_eq!(authors_info.title(), "Author");
+        assert_eq!(authors_info_2.title(), "Authors");
+    }
+
+    #[test]
+    fn test_author_info_value() {
+        let author = Author::new(
+            "John Doe".into(),
+            "john.doe@email.com".into(),
+            1500,
+            2000,
+            true,
+        );
+
+        let author2 = Author::new(
+            "Roberto Berto".into(),
+            "bertolone2000@email.com".into(),
+            240,
+            300,
+            false,
+        );
+
+        let authors_info = AuthorsInfo {
+            info_color: DynColors::Rgb(255, 0, 0),
+            authors: vec![author.clone()],
+        };
+        let authors_info_2 = AuthorsInfo {
+            info_color: DynColors::Rgb(255, 0, 0),
+            authors: vec![author, author2],
+        };
+
+        println!("{}", authors_info);
+        println!("{}", authors_info_2);
+
+        assert_eq!(
+            authors_info.value(),
+            "\u{1b}[38;2;255;0;0m75% John Doe <john.doe@email.com> 1500\u{1b}[39m"
+        );
```

Comment:
See comments like https://github.com/o2sh/onefetch/pull/827#discussion_r996912875

Test that the expected *plain text* is there with `.contains` instead of including the color formatting in the expected value.

Reply:
BTW, if you wanna to check the formatting without losing in readability - by adding a sequence of ANSI escape codes - you could just do :

```rust
  assert_eq!(
      authors_info.value(),
      "75% John Doe <john.doe@email.com> 1500"
          .color(DynColors::Rgb(255, 0, 0))
          .to_string()
  );

```

Reply:
Great idea, thanks for bringing that up!